integrations-and-supported-tools/pytorch/scripts/Neptune_Pytorch_Support_more_options.py

```
@@ -1,158 +1,151 @@
-from numpy.random import permutation
-import torch
-import torch.nn as nn
-import torch.optim as optim
-import torchvision
-from torchvision import datasets, models, transforms
-import neptune.new as neptune
-import numpy as np
-from neptune.new.types import File
-import torch.nn.functional as F
-
-
-# Step 1: Initialize Neptune and create new Neptune Run
-run = neptune.init(project="common/pytorch-integration", tags='More options script', api_token="ANONYMOUS", source_files=["*.py"])
-
-
-# Experiment Config
-data_dir = "data/CIFAR10"
-data_tfms = {
-    "train": transforms.Compose([
-        transforms.RandomHorizontalFlip(),
-        transforms.ToTensor(),
-        transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-    ]),
-    "val": transforms.Compose([
-        transforms.ToTensor(),
-        transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-        
-    ])
-}
-
-params = {
-    "lr": 1e-2,
-    "bs": 128,
-    "input_sz": 32 * 32 * 3,
-    "n_classes": 10,
-    "model_filename": "basemodel",
-    "device": torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-}
-
-# Model & Dataset
-class BaseModel(nn.Module):
-    def __init__(self, input_sz, hidden_dim, n_classes):
-        super(BaseModel, self).__init__()
-        self.main = nn.Sequential(
-            nn.Linear(input_sz, hidden_dim*2),
-            nn.ReLU(),
-            nn.Linear(hidden_dim*2, hidden_dim),
-            nn.ReLU(),
-            nn.Linear(hidden_dim, hidden_dim//2),
-            nn.ReLU(),
-            nn.Linear(hidden_dim//2, n_classes)
+def main():
```

@SiddhantSadangi same here, let it fail